bot/src/bot.js

```diff
@@ -28,6 +28,25 @@ function saveUpdate(db, author, channel, text) {
   });
 }
 
+const createDateRe = /create a report for ([0-9]{4}-[0-9]{2}-[0-9]{2})/;
```

Comment:
I think I'd prefer to avoid using slash for command.  Let's try to keep it natural-speech.